### DISCUSSION TITLE
Disable user access to ui if rbac not defined.

### DISCUSSION THREAD

**Discussion by riteshnanda09** [argoproj/argo-cd#11623] [General]
I am using Google oauth as an integration. I have added rbac for the users whom i want access to , also disabled default role in `argocd-rbac-cm` by adding `policy.default: role:''` , but still users are able to login once authorized by google oauth though they don't have any permissions. 

Is it possible to disable users to be not able to login to ui , if they don't have any permissions.

**Comment by ibuiltthis** [comment]
You might want to refer to: https://github.com/argoproj/argo-cd/issues/8742
